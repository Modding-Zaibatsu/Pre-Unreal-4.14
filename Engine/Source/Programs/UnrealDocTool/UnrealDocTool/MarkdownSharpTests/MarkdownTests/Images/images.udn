![Alt text](ue4.png)

![Alt text](ue4.png "Optional title")

![Alt text](ue4.png)(w:123 h:456 a:left convert:false type:jpg quality:100 fill:#DEADBE)