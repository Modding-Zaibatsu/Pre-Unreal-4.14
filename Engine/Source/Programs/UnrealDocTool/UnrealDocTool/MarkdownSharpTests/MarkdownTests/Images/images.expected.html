<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
	<head>
	</head>
	<body>
		<p><img alt="Alt text" src="./../../images\Temp\ue4.Jpeg" /></p>
		<p><img alt="Alt text" title="Optional title" src="./../../images\Temp\ue4.Jpeg" /></p>
		<p><img alt="Alt text" width="123" height="456" style="float: left;" src="./../../images\Temp\ue4.Png" /></p>
	</body>
</html>
